### PR TITLE
Update smartracks-minimal-image

### PR DESCRIPTION
**Pull request by HatsyRei**
- Remove wifi related packagegroups.
- Create packagegroup-ni-cli, migrate smartracks related packages into packagegroup-ni-cli.

### CODE REVIEW

File: recipes-images/images/packagegroup-ni-cli.bb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+"
+
+RDEPENDS_packagegroup-tdx-cli = "\
+    packagegroup-base-ni-cli \
+    packagegroup-devel-ni-cli \
+"
+
+SUMMARY_packagegroup-base-ni-cli = "Packages preinstalled into SmartRacks RCU image"
+RRECOMMENDS_packagegroup-base-ni-cli = "\
+    i2c-tools \
```
Comment (karhin1990) [inline recipes-images/images/packagegroup-ni-cli.bb]: This is a dev tool, and it seems has already included by [toradex packagegroup](https://git.toradex.com/cgit/meta-toradex-demos.git/tree/recipes-images/images/packagegroup-tdx-cli.bb?h=dunfell-5.x.y#n63), shall we remove it?

File: recipes-images/images/smartracks-minimal-image.bb [hunk 2]
```diff
@@ -31,15 +31,13 @@ IMAGE_LINGUAS = "en-us"
...
     packagegroup-boot \
     packagegroup-basic \
     packagegroup-base-tdx-cli \
     packagegroup-machine-tdx-cli \
-    packagegroup-wifi-tdx-cli \
-    packagegroup-wifi-fw-tdx-cli \
+    packagegroup-base-ni-cli \
     udev-extraconf \
     ${CONMANPKGS} \
     ${SMARTRACKSPKGS} \
```
Comment (karhin1990) [inline recipes-images/images/smartracks-minimal-image.bb]: This can be removed

**Review by karhin1990 — APPROVED** [review]

File: recipes-images/images/packagegroup-ni-cli.bb (hunk 1 above)
Comment (HatsyRei) [reply recipes-images/images/packagegroup-ni-cli.bb]: Removed.

File: recipes-images/images/smartracks-minimal-image.bb (hunk 2 above)
Comment (HatsyRei) [reply recipes-images/images/smartracks-minimal-image.bb]: Removed.

File: recipes-images/images/smartracks-minimal-image.bb [hunk 3]
```diff
@@ -31,18 +31,15 @@ IMAGE_LINGUAS = "en-us"
 #IMAGE_LINGUAS = "de-de fr-fr en-gb en-us pt-br es-es kn-in ml-in ta-in"
 
 CONMANPKGS ?= "connman connman-plugin-loopback connman-plugin-ethernet connman-plugin-wifi connman-client"
```
Comment (karhin1990) [inline recipes-images/images/smartracks-minimal-image.bb]: Shall we remove `connman-plugin-wifi` as well?

Comment (HatsyRei) [reply recipes-images/images/smartracks-minimal-image.bb]: Yep, I overlooked this one.

**Review by karhin1990 — APPROVED** [review]

**Pull request merged**